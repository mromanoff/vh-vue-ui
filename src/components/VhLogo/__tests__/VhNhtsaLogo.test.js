import { shallowMount } from '@vue/test-utils';
import VhNhtsaLogo from '../VhNhtsaLogo.vue';

describe('VhNhtsaLogo.vue', () => {
  let wrapper;

  it('renders default VhNhtsaLogo', () => {
    wrapper = shallowMount(VhNhtsaLogo);

    expect(wrapper).toMatchSnapshot();
  });
});
