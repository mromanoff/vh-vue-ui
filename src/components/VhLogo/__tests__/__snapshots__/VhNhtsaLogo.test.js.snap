// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VhNhtsaLogo.vue renders default VhNhtsaLogo 1`] = `<div><img src="https://www.vehiclehistory.com/images/nhtsa.png" alt="NHTSA Logo" class="NhtsaLogo"></div>`;
